src/status_im/contexts/chat/messenger/composer/effects.cljs

```diff
@@ -203,12 +198,10 @@
 (defn use-images
   [{:keys [sending-images? input-ref]}
    {:keys [text-value maximized?]}
-   {:keys [container-opacity height saved-height]}
+   {:keys [height saved-height]}
    {:keys [images]}]
   (rn/use-effect
    (fn []
-     (when images
-       (reanimated/animate container-opacity 1))
```

Comment:
Is the container's opacity being animated on adding images somewhere else? (and have you tested adding images, then existing and entering the chat screen)

Reply:
Yes, as I mentioned in above comment all animations for container-opacity is handled in composer.js file at one place. Now we don't need to handle this manually.

> and have you tested adding images, then existing and entering the chat screen

Double checked just now, working fine. Thank you for reminding.

Reply:
I also checked this case and it looks fine for me